subsys/net/l2/wifi/wifi_mgmt.c

```diff
@@ -343,3 +343,37 @@ void wifi_mgmt_raise_twt_sleep_state(struct net_if *iface,
 					iface, INT_TO_POINTER(twt_sleep_state),
 					sizeof(twt_sleep_state));
 }
+
+static int wifi_set_listen_intvl(uint32_t mgmt_request, struct net_if *iface,
```

Comment:
Please use full interval, I don't think we are saving anything with the short form.